rust-src/concordium_base/src/common/types.rs

```diff
@@ -303,7 +303,19 @@ pub struct TransactionTime {
 }
 
 impl TransactionTime {
+    /// Construct a timestamp from seconds since the unix epoch.
     pub fn from_seconds(seconds: u64) -> Self { Self { seconds } }
+
+    /// Construct a timestamp that is the given amount of seconds in the future.
+    pub fn seconds_after(seconds: u32) -> Self {
```

Comment:
I assume the `u32` is sufficient for most years, but please consider adding a comment about it.

Reply:
Well 32-bits will fit setting expirty up to 136 years in the future. I don't think this needs a comment for the user.

I chose u32 because this avoids any overflows.

Reply:
Ok, sounds good.